euth/flashpoll/templates/euth_flashpoll/includes/project_form_result_flashpoll_tab.html

```diff
@@ -5,8 +5,8 @@
 <script type="text/javascript">
 
 var apiKey = '{% api_key %}';
-var pollresult = {{pollresult|get_simplejson|safe}};
-var poll = {{poll|get_simplejson|safe}};
+var pollresult = JSON.parse('{{ pollresult|get_simplejson|escapejs }}');
+var poll = JSON.parse('{{ poll|get_simplejson|escapejs }}');
```

Comment:
I am not entirely sure that this is secure. The documentation is a bit unclear. https://stackoverflow.com/questions/23331988/#answer-23341128 suggests that it is, but data attributes are still preferable.